OpenCV-4-5-5.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
We should probably add a reference to where this script came from.

Reply:
absolutely. no clue where that is. i got it from u lol

for now it's kinda just for us and maybe we'll source a generic one that should run on our setup script or somethin